examples/svg-components/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "svg-components",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "beta"
+  },
+  "devDependencies": {
+    "babel-plugin-inline-react-svg": "^0.2.0"
```

Comment:
Yeah, I had hopes for that one too. Unfortunately, I don't think it'll work, since webpack isn't used to build the server files :(